### PR TITLE
Some programs for python beginners

### PR DESCRIPTION
**Pull request by ghost**
Adding some code examples for pythin beginners . Will be happy to help you with more . Looking forward to contribute more. Thank You . I hope you will add them .

### CODE REVIEW

File: countLinesFile.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+rax = open('Hello.txt','r')
+count = 0
+for nat in rax:
+    count = count + 1
+print('Total Lines : ',count)
```
Comment (Akuli) [inline countLinesFile.py:5]: Can you put this program to `basics/files.md`, near other code examples of looping over a file, and add some text explaining what this code does?

File: findLargest.py [hunk 2]
```diff
@@ -0,0 +1,6 @@
+defNo = 0
+print('Before :' , defNo)
+for num in [74,54,63,98,41,28,74,63,85,98]:
+    if num > defNo:
+        defNo = num
+print('Largest Number: ',defNo)
```
Comment (Akuli) [inline findLargest.py:6]: Can you turn this into a "fix this broken code" exercise and put it to `loops.md`?

File: notify.py [hunk 3]
```diff
@@ -0,0 +1,3 @@
+import win10toast
+toaster = win10toast.ToastNotifier()
+toaster.show_toast('Rax','Its my PC',duration = 7)
```
Comment (Akuli) [inline notify.py:3]: Can you create an exercises section to `basics/modules.md` with an exercise saying e.g. "create a program that shows a notification"? This code should then go to `basics/answers.md`.

**Review by Akuli — CHANGES_REQUESTED** [review]

**Comment by tarusharora03** [comment]
Can you wait for some time. I am creating some more programs regarding python for beginners. Don't close this pull request i will commit all the changes to it in some time.
I hope you understand and Thank you for your feedback.

**Comment by ghost** [comment]
Yeah, sure i will commit changes as per requested. I am creating some more programs regarding python for beginners and i will add them in this pull request. Just don't close it i will commit new changes in itself. Thank you for your feedback. Hope you liked the programs.

**Pull request closed without merging**